app/scripts/modules/google/src/serverGroup/configure/wizard/zones/targetShapeSelector.directive.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
```

Comment:
Since this is a new file we're adding, can we make it typescript? Then can get rid of the `'use strict'` as well.

Reply:
Done.